dbt/models/phmsagas/_core_phmsagas__yearly_distribution_misc/schema.yml

```diff
@@ -15,7 +15,6 @@ sources:
           - dbt_expectations.expect_compound_columns_to_be_unique:
               arguments:
                 column_list:
-                  - report_date
```

Comment:
`report_date` was not listed in the expected natural PK in the metadata annotations, doesn't show up in the other tables, and the PK was unique without it, so I removed it here.